source/MRMesh/MRMeshLoadSettings.h

```diff
@@ -13,6 +13,7 @@ struct MeshLoadSettings
     int* skippedFaceCount = nullptr; ///< counter of skipped faces (faces than can't be created)
     int* duplicatedVertexCount = nullptr; ///< counter of duplicated vertices (that created for resolve non-manifold geometry)
     ProgressCallback callback;       ///< callback for set progress and stop process
+    AffineXf3f* xf = nullptr; ///< transform for the loaded mesh
```

Comment:
mb put it before `ProgressCallback` (usually `ProgressCallback` is last parameter)

Reply:
Fixed.